### PR TITLE
Documentation updates for v3.1

### PR DESCRIPTION
**Pull request by bgeltz**
- Relates to #3384.

### CODE REVIEW

File: docs/source/overview.rst [hunk 1]
```diff
@@ -13,47 +13,60 @@ For in-depth information see: :doc:`service` or :doc:`runtime`.
...
+have been included in their `v0.22.0 release
+<https://github.com/spack/spack/tree/v0.22.0>`_.  These recipes currently allow
+for building both the v3.0.1 release of GEOPM and our main development branch.
 
-For users leveraging the Aurora/Sunspot computing resources at the Argonne
-Leadership Computing Facility (ALCF), the GEOPM Service is installed in the base
-compute image.  The userspace components of both the Service and the Runtime
-are available via environment modules.  These components can be loaded on the
-command-line via:
+For deploying GEOPM's layers to a compute image, a typical configration would
```
Comment (cmcantalupo) [inline docs/source/overview.rst]:
```suggestion
For deploying GEOPM's layers to a compute image, a typical configuration would
```

**Review by cmcantalupo — DISMISSED** [review]

File: docs/source/overview.rst [hunk 2]
```diff
@@ -13,47 +13,60 @@ For in-depth information see: :doc:`service` or :doc:`runtime`.
...
+<https://github.com/spack/spack/tree/v0.22.0>`_.  These recipes currently allow
+for building both the v3.0.1 release of GEOPM and our main development branch.
 
-For users leveraging the Aurora/Sunspot computing resources at the Argonne
-Leadership Computing Facility (ALCF), the GEOPM Service is installed in the base
-compute image.  The userspace components of both the Service and the Runtime
-are available via environment modules.  These components can be loaded on the
-command-line via:
+For deploying GEOPM's layers to a compute image, a typical configuration would
+be to have a system install of the service RPMs baked into the compute image,
```
Comment (asmaalrawi) [inline docs/source/overview.rst]:
```suggestion
be to have a system install of the service RPMs baked into the compute image
```

File: docs/source/overview.rst [hunk 3]
```diff
@@ -13,47 +13,60 @@ For in-depth information see: :doc:`service` or :doc:`runtime`.
...
+For GEOPM v3.0.1, system install ``geopm-service``, ``libgeopmd2``, and ``python3-geopmdpy``
 
-    $ module load geopm
+For GEOPM v3.1, system install ``geopm-service``, ``libgeopmd2``, ``python3-geopmdpy``, and ``python3-geopmdpy-doc``
 
-To list all of the available GEOPM versions issue:
+In order to build with spack this way, ``geopm-service`` must be configured as an
+external package in ``~/.spack/packages.yaml``:
 
 .. code-block:: bash
```
Comment (cmcantalupo) [inline docs/source/overview.rst]: bash -> yaml?

File: docs/source/index.rst [hunk 4]
```diff
@@ -17,44 +17,59 @@ Users can monitor their system's energy and power consumption, and safely
...
+   :target: https://github.com/geopm/geopm
+   :width: 200
+   :alt: View on Github
+
+----
+
+With GEOPM, a system administrator can:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|:closed_lock_with_key:| Provide secure access to hardware telemetry and controls
```
Comment (cmcantalupo) [inline docs/source/index.rst]:
```suggestion
|:closed_lock_with_key:| Enable users with fine-grained access management
```

File: docs/source/overview.rst [hunk 5]
```diff
@@ -764,15 +777,15 @@ at the ``board`` domain will result in each ``package`` receiving a limit of
...
 
 ----
 
-|:straight_ruler:| Use the Runtime to Measure Application Performance
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+|:straight_ruler:| Measure Performance
+--------------------------------------
 
 The GEOPM Runtime offers capabilities for collecting telemetry throughout an
 application's execution. If you're aiming to measure a particular segment of an
```
Comment (asmaalrawi) [inline docs/source/overview.rst]:
```suggestion
application's execution. If you want to measure a particular segment of an
```

File: docs/source/index.rst [hunk 6]
```diff
@@ -17,44 +17,59 @@ Users can monitor their system's energy and power consumption, and safely
...
   - Grant per-user or per-group access to individual metrics and controls, even
     when their underlying interfaces do not offer fine-grained access control
+|:safety_vest:| Provide safe access to hardware settings
   - Ensure that user-driven changes to hardware settings are reverted when the
     user's process session terminates
+|:building_construction:| Develop new plugins
```
Comment (cmcantalupo) [inline docs/source/index.rst]:
```suggestion
|:building_construction:| Extend the open-core framework
```

**Review by asmaalrawi — COMMENTED** [review]
Very minor nit.

File: docs/source/overview.rst (hunk 5 above)
Comment (asmaalrawi) [reply docs/source/overview.rst]: Colloquialism might be difficult for non-native English speakers.

File: docs/source/index.rst [hunk 7]
```diff
@@ -17,44 +17,59 @@ Users can monitor their system's energy and power consumption, and safely
...
   - Develop your own platform-specific monitor and control interfaces through
     the extensible plugin architecture
 
-**With GEOPM, an end user can:**
+With GEOPM, an end user can:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+|:microscope:| Gather hardware telemetry and set hardware configuration
```
Comment (cmcantalupo) [inline docs/source/index.rst]:
```suggestion
|:microscope:| Monitor telemetry and configure settings on the platform
```

**Review by cmcantalupo — APPROVED** [review]

**Pull request merged**